changelog/6.0.0_2024-06-19/activity-service.md

```diff
@@ -2,4 +2,5 @@ Enhancement: Activitylog Service
 
 Adds a new service `activitylog` which stores events (activities) per resource. This data can be retrieved by clients to show item activities
 
+https://github.com/owncloud/ocis/pull/9360
```

Comment:
This PR is "fix: thumbnail max input height"
Accidentally added here?

Reply:
off-by-one error :smile: Thanks!